core/tests/protractor_utils/workflow.js

```diff
@@ -119,12 +119,22 @@ var createAndPublishExploration = function(
 
 // Role management (state editor settings tab)
 
+// Add a title to the exploration
+var addTitle = function(){
```

Comment:
We already have a function for setting the title for the exploration.
(See `core/tests/protractor_utils/ExplorationEditorSettingsTab.js` --> this.setTitle)

Reply:
Yeah, but I did not know exactly how to import it and run it, and adding the title was oneline, (check if title exists and add), so I did not add it.